cocos/scripting/js-bindings/jswrapper/v8/ScriptEngine.cpp

```diff
@@ -461,8 +461,8 @@ namespace se {
         if(!jitSupported()) {
             flags.append(" --jitless");
         }
-#elif CC_TARGET_PLATFORM == CC_PLATFORM_OPENHARMONY
-        flags.append(" --jitless");
+#elif (CC_TARGET_PLATFORM == CC_PLATFORM_OPENHARMONY)
```

Comment:
Can add it in above if, like 
```
#if (CC_TARGET_PLATFORM == CC_PLATFORM_IOS || CC_TARGET_PLATFORM == CC_PLATFORM_OPENHARMONY)
```

Reply:
That's it, iOS still has jitSupported judgment

Reply:
Ok